### ISSUE TITLE
Can't authorize App

### ISSUE THREAD

**Issue by pa-philos** [rmuilwijk/MauticFBAdsLeadAdsBundle#6]
I think FB has made a silent update or it's not supporting the SDK 2.9 anymore. It gives me this error:

Can't Load URL: The domain of this URL isn't included in the app's domains. To be able to load this URL, add all domains and subdomains of your app to the App Domains field in your app settings.

I'm pretty sure I did everything right, but it still isn't working.

![image](https://user-images.githubusercontent.com/829823/41503202-99b428b2-71a3-11e8-96d5-f194b41ed7c5.png)

**Comment by YosuCadilla** [comment]
Same problem here, I think that, as you mentioned, FB changed A LOT in how apps and app security needs to be managed now.

Did you find a solution @Shirkit ?

**Comment by pa-philos** [comment]
@YosuCadilla No, but I think I found the root cause. FB dropped the 2.9 PHP API, so it's likely it's using an old way to try to authenticate.

**Comment by YosuCadilla** [comment]
Guys I finally got past this error msg (just to fall to the next one).
I had to review all my configuration like a 100 times until I finally got it right.
Not saying the plugin works or not, just saying this error can be sorted out on the FB settings side.

**Comment by YosuCadilla** [comment]
The new error is this one:
https://github.com/rmuilwijk/MauticFBAdsLeadAdsBundle/issues/7

**Comment by dsp76** [comment]
Actually the error message of Facebook is missleading. It likely doesn't complain about the domain missing in the app settings (probably added already). But about the URL not being entered into "Valid OAuth Redirect URIs" under Client OAuth Settings for Facebook Login.
